src/pages/extensions/iter8/Iter8ExperimentDetails/ExperimentCreatePage.scss

```diff
@@ -0,0 +1,7 @@
+.pf-c-switch__toggle {
```

Comment:
I'm afraid this breaks the style in other pages:
![image](https://user-images.githubusercontent.com/1662329/88651416-e9ef4f00-d0c9-11ea-8972-d43a43aa096c.png)

So, I'd suggest to skip the change of color.
I know we have a pending review of the design of this form later, perhaps it's something we can adjust with the other Istio forms.